packages/fondue/src/components/RichTextEditor/serializer/utils/serializeNodeToHtmlRecursive.ts

```diff
@@ -126,7 +126,7 @@ const MapNodeTypesToHtml: { [key: string]: ({ ...args }: Arguments) => string }
             getLiStyles(node, styles),
         )}">${children}</li>`,
     [ELEMENT_LIC]: ({ classNames, children, node }) =>
-        `<p dir="auto" class="${classNames} ${getLicElementClassNames(node)}"><span>${children}</span></p>`,
+        `<p dir="auto" class="${classNames} ${getLicElementClassNames(node)}"><span class="tw-min-w-0">${children}</span></p>`,
```

Comment:
Better to put the classes into a variable and use it in both places (plugin and serializer)

https://github.com/Frontify/fondue/blob/79abb46358c17ca2f750f3b76b81ac4fe0d887d6/packages/fondue/src/components/RichTextEditor/Plugins/ListPlugin/ListItemContentMarkupElement.tsx#L21

Reply:
Thanks, adjusted